src/hardware/mouse/mouseif_virtual_machines.cpp

```diff
@@ -70,21 +70,20 @@ static constexpr uint32_t ABS_UPDATED = 4;            // tells that new pointer
                                                       // position is available
 static constexpr uint32_t ABS_NOT_UPDATED = 0;
 
-static bool updated = false;  // true = mouse state update waits to be picked up
-static VMwareButtons buttons; // state of mouse buttons, in VMware format
-static uint16_t scaled_x = 0x7fff; // absolute position scaled from 0 to 0xffff
-static uint16_t scaled_y = 0x7fff; // 0x7fff is a center position
-static int8_t wheel_counter = 0;      // wheel movement counter
+static bool raw_input = true;       // true = no host mouse acceleration pre-applied
+static bool is_mapped = false;      // true = physical mouse is mapped to this interface
+static bool updated   = false;      // true = mouse state update waits to be picked up
+static VMwareButtons buttons;       // state of mouse buttons, in VMware format
+static uint16_t scaled_x  = 0x7fff; // absolute position scaled from 0 to 0xffff
+static uint16_t scaled_y  = 0x7fff; // 0x7fff is a center position
+static int8_t   counter_w = 0;      // wheel movement counter
 
 static float pos_x = 0.0f;
 static float pos_y = 0.0f;
 
-// Speed measurement
-static auto time_start      = std::chrono::steady_clock::now();
-static auto pic_ticks_start = PIC_Ticks;
-static float distance       = 0.0f; // distance since last measurement
-
-static float speed = 0.0f;
+// Multiply scale by 0.02f to put acceleration_vmm in a reasonable
+// range, similar to sensitivity_dos or sensitivity_vmm)
+static MouseSpeedCalculator speed_xy(0.02f * mouse_predefined.acceleration_vmm);
```

Comment:
Suggest lifting out the 0.2 scalar to a `constexpr float`.

Reply:
OK, changed.